dnf5/main.cpp

```diff
@@ -415,6 +415,14 @@ void RootCommand::set_argument_parser() {
     installroot->link_value(&config.installroot());
     global_options_group->register_argument(installroot);
 
+    auto use_host_environment = parser.add_new_named_arg("use-host-environment");
```

Comment:
Just a small nit:  "host environment" seems like some /home directory was shared, or some set of environment variables are shared.  Since this is about configuration, I'd spell it "use-host-config".

Reply:
It looks like that the name of the option is the only thing to discuss.

Reply:
I agree with @praiskup, I think `--use-host-config` is clearer. Both the configuration file and the reposdir are affected by this flag, so in that sense it is more than just "configuration", but in this case I think it's a better name. "Environment" is just such an overloaded term.

Reply:
Then please  feel free to rename it. `--use-host-config` and configuration option `use-host-config`.

Reply:
Looking at our conventions, the option should rather be with underscores `use_host_config`, right?

Reply:
Correct, my mistake
Then please feel free to rename it. `--use-host-config` and configuration option `use_host_config`.